cogs/helpers.py

```diff
@@ -53,11 +56,57 @@
 CURRENCY_TEMPLATE = "http://www.xe.com/currencyconverter/convert/" \
                     "?Amount={}&From={}&To={}"
 
+try:
+    os.mkdir('./pickles')
```

Comment:
What is the current state of logging? It might be time for us to make periodic backups of assets like this and logs. (Not a change for you to make, but something for us to think about).